hadoop-common-project/hadoop-common/src/site/markdown/filesystem/filesystem.md

```diff
@@ -629,10 +633,18 @@ The result is `FSDataOutputStream`, which through its operations may generate ne
  clients creating files with `overwrite==true` to fail if the file is created
  by another client between the two tests.
 
-* S3A, Swift and potentially other Object Stores do not currently change the FS state
+* S3A, Swift and potentially other Object Stores do not currently change the `FS` state
 until the output stream `close()` operation is completed.
-This MAY be a bug, as it allows >1 client to create a file with `overwrite==false`,
- and potentially confuse file/directory logic
+This is a significant difference between the behavior of object stores
+and that of filesystems, as it allows >1 client to create a file with `overwrite==false`,
```

Comment:
nit: You used `overwrite=false` below. I'm assuming the double-equals was just copy-paste, but maybe correct to stay consistent?

Reply:
changed to ==; also replace >1 with &gt; to keep everything happy